mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb

```diff
@@ -35,7 +40,10 @@ resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
           name  = "baz"
           value = "quux"
         }
+        # [END cloudrun_service_configuration_env_var]
 
+        # [START cloudrun_service_configuration_limit_memory]
+        # [START cloudrun_service_configuration_limit_cpu]
```

Comment:
Does enclosure order matter ?

`cloudrun_service_configuration_limit_memory` seems to contain `cloudrun_service_configuration_limit_cpu` but ends before `cloudrun_service_configuration_limit_memory`

Reply:
Nope!